.github/workflows/gradle.yml

```diff
@@ -0,0 +1,16 @@
+name: Java CI
+
+on: [push]
```

Comment:
This should include pull-request because someone might make a pull request from a branch in a fork.